watertap/data/techno_economic/suboxic_activated_sludge_process.yaml

```diff
@@ -0,0 +1,89 @@
+default:
+  energy_electric_flow_vol_inlet:
+    value: 0.5
+    units: kWh/m^3
```

Comment:
This is likely the most important aspect for the unit in comparison to the conventional approach. In reality, we would calculate the compressor work required to supply the amount of air/oxygen going to the tank, so I'd think we'd want to approach this similar to how we have applied `pump_electricity` to calculate pump work. 

Now, if this is meant as an early first step due lack of info, then I would say that we should at least compare this value with what is listed for other related zero-order units, like `conventional_activated_sludge` and `aeration_basin` (or `aeration_tank` ?)

Reply:
We don't have information for the `pump_electricity` right now, but their original data sheet does provide an alternative electricity metrics based on daily oxygen demand instead of inlet flow. But I check the `conventional_activated_sludge` and `aeration_basin` models, they also both use the electricity method based on the inlet flow. The `energy_electric_flow_vol_inlet` for `conventional_activated_sludge` is 0.14 kWh/m3 and for `aeration_basin` is 0.41226735 kWh/m3

Reply:
Then at a minimum, I would expect that the default value used here should be less than either of those for `conventional_activated_sludge` or `aeration_basin` (assuming the default values for the latter two are relevant in the first place).

Reply:
The value is provided by the Tanja's team, they said their major advantage is their advanced control technology, which use less money than the conventional control. I think they might need to consider relating it to the energy consumption.

Reply:
OK, I would say that any issue here can be resolved after iteration with the team. To my understanding, a key advantage should be lower energy consumption in comparison to conventional approach, so it's something that should be revisited later.

Reply:
Yeah, I fully agree your thought. I think the advantages of their control strategy should be somehow adressed by low energy consumption. I will make a note and revist this value.